### PR TITLE
Update database models 

### PR DESCRIPTION
**Pull request by kaashmonee**
This PR introduces several changes in the database models in order to address #65. 

The changes include: 
* A Django admin interface, where an administrative user can view, modify, and delete other users, hospitals, and hospital properties. 
* A new `Account` model to store user information. To ensure that we can distinguish between different users, there are is one singular `Account` model that stores basic user information and 3 other models that represent the different kinds of users that we have. (As mentioned in #65.) 
* Migrations to reflect all the previous changes.
* Updated `Hospital` that now contains the option to view and modify all of the hospital's resources.

### CODE REVIEW

File: accounts/serializers.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    class Meta:
+        model = Account
+        fields = ["email", "password", "date_joined", "last_login", "hospital",]
+
+
+    def create(self, validated_data):
+        cleaned_email = validated_data["email"]
+        cleaned_password = validated_data["password"]
+
+        acct = Account
```
Comment (bl-nero) [inline accounts/serializers.py:21]: What does this line do?

File: hospitals/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.0.6 on 2020-05-28 02:39
+# Generated by Django 3.0.7 on 2020-07-26 03:24
```
Comment (bl-nero) [inline hospitals/migrations/0001_initial.py:1]: How will our existing databases react to changing an initial migration?

File: accounts/admin.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
```
Comment (bl-nero) [inline accounts/admin.py:1]: If this is an admin UI, this is gonna clash with what @naijauser started preparing. Nnamdi, please make sure that you don't both do the same thing.

Comment (kaashmonee) [reply accounts/admin.py:1]: I believe this should still be fine. From what I saw, @naijauser was building the hospital admin dashboard, which I think we could still use. This administrator tool gives us access to all the hospitals, its users, patients, doctors, etc., so I imagine it being be a tool for admins/developers of the Infomed site (us).

File: hospitals/migrations/0001_initial.py (hunk 2 above)
Comment (kaashmonee) [reply hospitals/migrations/0001_initial.py:1]: I think it should be fine --- unless, is there any data currently in our database that we want to keep?

File: accounts/serializers.py (hunk 1 above)
Comment (kaashmonee) [reply accounts/serializers.py:21]: Ahh, I think this must've snuck in from another development branch. This is an incomplete (still in progress) serializer for the Account model. It should be removed for this PR.

File: hospitals/migrations/0001_initial.py (hunk 2 above)
Comment (bl-nero) [reply hospitals/migrations/0001_initial.py:1]: @kaashmonee I guess I've just fallen into this trap. I had to drop my database and recreate it from scratch, and given the fact that we don't have any script to create a hospital, and the procedure is not documented, it was a rough ride. We should pay attention not to break our own local development environments in the future.

**Pull request merged**